presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergQueryRunner.java

```diff
@@ -197,7 +203,12 @@ public static DistributedQueryRunner createIcebergQueryRunner(
         queryRunner.installPlugin(new TpcdsPlugin());
         queryRunner.createCatalog("tpcds", "tpcds");
 
-        queryRunner.installPlugin(new IcebergPlugin());
+        List<MBeanServer> mBeanServerList = queryRunner.getServers().stream()
+                .map(server -> {
+                    MBeanServer mBeanServer = MBeanServerFactory.newMBeanServer();
+                    server.installPlugin(new IcebergPlugin(mBeanServer));
+                    return mBeanServer;
+                }).collect(Collectors.toList());
```

Comment:
Nit: I'd move the JmxConnector and IcebergConnector creation to the same place so it's obvious they're sharing the same MBeanServer.

```suggestion
        queryRunner.getServers().forEach(server -> {
                    MBeanServer mBeanServer = MBeanServerFactory.newMBeanServer();
                    server.installPlugin(new IcebergPlugin(mBeanServer));
                    if (addJmxPlugin) {
                        server.installPlugin(new JmxPlugin(mBeanServer));
                    }
                });
```

Reply:
Good suggestion, fixed, the code looks more concise. Please take a look when convenient, thanks a lot.